### PR TITLE
fix image wcs header

### PR DESCRIPTION
**Pull request by dsavchenko**

### CODE REVIEW

File: oda_api/data_products.py [hunk 1]
```diff
@@ -763,7 +763,7 @@ def from_arrays(cls,
                                  meta_data = meta_data,
                                  data_header = data_header,
                                  hdu_type = 'bintable',
-                                 name = 'LIGHTCURVE')],
+                                 name = 'LC')],
```
Comment (dsavchenko) [inline oda_api/data_products.py:766]: This is unrelated. Anyway, the name is arbitrary

**Review by volodymyrss — APPROVED** [review]
